nilearn/plotting/html_connectome.py

```diff
@@ -134,13 +148,51 @@ def view_connectome(adjacency_matrix, coords, threshold=None,
 
     """
     connectome_info = _get_connectome(
-        adjacency_matrix, coords, threshold=threshold, cmap=cmap,
+        adjacency_matrix, node_coords, threshold=edge_threshold, cmap=edge_cmap,
         symmetric_cmap=symmetric_cmap)
     connectome_info["line_width"] = linewidth
-    connectome_info["marker_size"] = marker_size
+    connectome_info["marker_size"] = node_size
     return _make_connectome_html(connectome_info)
 
 
+def _warn_deprecated_params_view_connectome(kwargs):
+    all_deprecated_params = {'coords': 'node_coords',
+                             'threshold': 'edge_threshold',
+                             'cmap': 'edge_cmap',
+                             'marker_size': 'node_size',
+                             }
+    used_deprecated_params = set(kwargs.keys()
+                                 ).intersection(all_deprecated_params.keys())
+    for deprecated_param_ in used_deprecated_params:
+        replacement_param = all_deprecated_params[deprecated_param_]
+        param_deprecation_msg = (
+            'The parameter "{}" will be removed in a future version of Nilearn.'
```

Comment:
Shouldn't we state which version ?

Reply:
For Python2 deprecation we hadn't so I didn't here either.
From what I understand we display deprecation warnings for 2 releases and start clarifying the feature etc will be removed in the next release.

I assume we will handle this similarly.
I am happy to add a specific version now, if you tell me which one it should be. Nilearn 0.7.0? Or do we include point releases in next version?

Reply:
I'd go for 0.7.0

Reply:
the sooner we make the change the less code it will break